.changes/next-release/Bug Fix-3b5c1306-6cbd-4592-a73e-bc92120ddcf3.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Shorten critical path between presigned url creation and usage to lower chance of url expiry"
```

Comment:
Please review the changelog guidlines: https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog

Reply:
Done.